README.md

```diff
@@ -102,9 +112,9 @@ This specification supports different scenarios for exposing bindable data. Belo
       
 3. Regular k8s resources (Ingress, Route, Service, Secret, ConfigMap etc)  - An annotation in the corresponding Kubernetes resources that maps the `status`, `spec` or `data` properties to their corresponding [binding data](#service-binding-schema). 
 
-All annotations used in CRDs in the above section can be used for regular k8s resources, as well.
+All annotations used in CRDs in the above section **MAY** be used for regular k8s resources, as well.
```

Comment:
I understand the keyword `MAY` has to be capitalized. 

But do we have to make it a `bold` text as well?

Reply:
Bold will make it stand out. I think it's better to use either bold or italic form. A couple of examples:
1. https://jsonapi.org/format/ (Bold)
2. https://www.w3.org/TR/json-ld11/ (Italic)